samples/cellular/location/sample.yaml

```diff
@@ -34,7 +34,8 @@ tests:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
```

Comment:
We'd need to add nrf9161dk_nrf9161_ns to all configurations of this file. Should I do that with a separate PR?

Reply:
Yes please that would be better, this PR is only for Wi-Fi parts.

Reply:
I plan to cover the document updates for the location sample as part of this PR. Should that be covered in the new PR?

Reply:
Preferably, as I would then need to add you as a colloborator.

Reply:
PR https://github.com/nrfconnect/sdk-nrf/pull/13048 for Location sample

Reply:
You can remove the changes in this file from this PR.